### PR TITLE
Add underscore to drop off window fields

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Updating front end  drop off window fields to include latest changes with underscores. If you hate me you'll make me capitalize the variables to conform to the underscore on the backend too. 

Blocked awaiting backend PR: https://github.com/conveyal/gtfs-lib/pull/380

### CODE REVIEW

File: lib/editor/actions/trip.js [hunk 1]
```diff
@@ -122,8 +122,8 @@ export function fetchTripsForCalendar (
               stopSequence: stop_sequence
               arrivalTime: arrival_time
               departureTime: departure_time
-              startPickupDropoffWindow: start_pickup_dropoff_window
-              endPickupDropoffWindow: end_pickup_dropoff_window
+              startPickupDropoffWindow: start_pickup_drop_off_window
```
Comment (miles-grant-ibigroup) [inline lib/editor/actions/trip.js]: It does look a bit silly doesn't it with the lower case o... I'll leave it up to the second reviewer to decide

**Review by miles-grant-ibigroup — APPROVED** [review]
Tested and working!

File: lib/editor/actions/trip.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/editor/actions/trip.js]: In datatools-ui we inconsistently use different casings, although `dropOff` seems to be used more often.
I would side with fixing the case here too to `start/endPickupDropOffWindow`.

**Review by binh-dam-ibigroup — APPROVED** [review]
Approving subject to addressing the casing comment that @miles-grant-ibigroup brought up.

**Pull request merged**